crates/bevy_ecs/src/world/deferred_world.rs

```diff
@@ -341,6 +341,21 @@ impl<'w> DeferredWorld<'w> {
         self.get_entity_mut(entities).unwrap()
     }
 
+    /// Provides split access to fetching entities and to queuing commands.
```

Comment:
I'd like a bit of explanation in this doc comments (ideally on a new line following a line break) about why this might ever by desirable.